libs/test/SmartStoreTest/src/com/salesforce/androidsdk/store/StoreConfigTest.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) 2017-present, salesforce.com, inc.
+ * All rights reserved.
+ * Redistribution and use of this software in source and binary forms, with or
+ * without modification, are permitted provided that the following conditions
+ * are met:
+ * - Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ * - Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ * - Neither the name of salesforce.com, inc. nor the names of its contributors
+ * may be used to endorse or promote products derived from this software without
+ * specific prior written permission of salesforce.com, inc.
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package com.salesforce.androidsdk.store;
+
+import android.content.Context;
+
+import com.salesforce.androidsdk.MainActivity;
+import com.salesforce.androidsdk.smartstore.app.SmartStoreSDKManager;
+import com.salesforce.androidsdk.smartstore.store.IndexSpec;
+import com.salesforce.androidsdk.smartstore.store.SmartStore;
+import com.salesforce.androidsdk.ui.LoginActivity;
+
+import org.json.JSONException;
+
+import java.util.List;
+
+public class StoreConfigTest extends SmartStoreTestCase {
+
+    private SmartStoreSDKManager sdkManager;
+    private SmartStore globalStore;
+    private SmartStore userStore;
+
+    @Override
+    public void setUp() throws Exception {
+        super.setUp();
+
+        SmartStoreSDKTestManager.init(getInstrumentation().getTargetContext(), store);
+        sdkManager = SmartStoreSDKTestManager.getInstance();
+        globalStore = sdkManager.getGlobalSmartStore();
+        userStore = sdkManager.getSmartStore();
+    }
+
+    @Override
+    protected String getEncryptionKey() {
```

Comment:
As mentioned above, should we remove this interface and its implementation and use `sdkManager.getEncryptionKey()` directly in all tests instead? Doesn't look like we care about the actual value of the encryption key in tests, so why bother setting it everywhere?

Reply:
SmartStoreTestCase builds a store directly without using a SDK manager.

Reply:
Gotcha.